### PR TITLE
Fix CalculateActualInterval() stuck in infinite loop (#758).

### PR DESCRIPTION
**Pull request by vhoehn**
Suggested fix for issue #758.

### CODE REVIEW

**Comment by objorke** [comment]
I added the example and updated the change log in #766

File: Source/OxyPlot/Axes/Axis.cs [hunk 1]
```diff
@@ -1593,6 +1593,16 @@ protected double CalculateActualInterval(double availableSize, double maxInterva
                 return maxIntervalSize;
             }
 
+            if (Math.Abs(maxIntervalSize) < Double.Epsilon)
```
Comment (objorke) [inline Source/OxyPlot/Axes/Axis.cs]: should use double

File: Source/OxyPlot/Axes/Axis.cs [hunk 2]
```diff
@@ -1593,6 +1593,16 @@ protected double CalculateActualInterval(double availableSize, double maxInterva
...
+            {
+                throw new ArgumentException("Maximum interval size cannot be zero.", "maxIntervalSize");
+            }
+
+            if (Math.Abs(range) < Double.Epsilon)
```
Comment (objorke) [inline Source/OxyPlot/Axes/Axis.cs]: double

**Comment by objorke** [comment]
I merged your change in #766

**Comment by alexdegroot** [comment]
This one is solved in 32-bit, but still reproducible in 64-bit.

**Pull request merged**